src/__tests__/opportunities.int.test.ts

```diff
@@ -53,48 +54,22 @@ describe('/opportunities', () => {
 		});
 
 		it('returns exactly one opportunity selected by id', async () => {
-			const opportunityIds: Result<{ id: number; title: string }> =
-				await db.query(`
-        INSERT INTO opportunities (
-          title,
-          created_at
-        )
-        VALUES
-          ( '🔥', '2525-01-02T00:00:01Z' ),
-          ( '✨', '2525-01-02T00:00:02Z' ),
-          ( '🚀', '2525-01-02T00:00:03Z' )
-        RETURNING id, title;
-      `);
-			logger.debug('opportunityIds: %o', opportunityIds);
-			const sparkleOpportunity = opportunityIds.rows.filter(
-				(it) => it.title === '✨',
-			)[0];
-			if (sparkleOpportunity === undefined) {
-				throw new Error(
-					'The sparkle opportunity could not be created so the test must fail.',
-				);
-			}
-			logger.debug('sparkleOpportunityId: %d', sparkleOpportunity.id);
-			await agent
-				.get(`/opportunities/${sparkleOpportunity.id}`)
+			await createOpportunity({ title: '🔥' });
+			await createOpportunity({ title: '✨' });
+			await createOpportunity({ title: '🚀' });
+
+			const response = await agent
+				.get(`/opportunities/2`)
 				.set(authHeader)
-				.expect(200, {
-					id: sparkleOpportunity.id,
-					createdAt: '2525-01-02T00:00:02.000Z',
-					title: '✨',
-				});
+				.expect(200);
+			expect(response.body).toEqual({
+				id: 2,
+				createdAt: expectTimestamp,
+				title: '✨',
+			});
 		});
 
 		it('returns 400 bad request when id is a letter', async () => {
-			await db.query(`
-        INSERT INTO opportunities (
-          title,
-          created_at
-        )
-        VALUES
-          ( 'This should not be returned', '2525-01-02T00:00:04Z' ),
```

Comment:
Why not keep at least one of these? I'm OK with leaving them out, just curious about the reason for the change.

Reply:
Setup does have a cost (minimal, but still there) -- the validity of an input is a the level of API definition / shouldn't depend on any given db state.  Given that, I figured why not go with the cheapest DB state!